lib/config.js

```diff
@@ -6,34 +6,25 @@ const path = require('path')
 
 module.exports = {
   AWS_INSTANCE_CHECK_URL: 'http://169.254.169.254/latest/dynamic/instance-identity/document/'
-, BUF_LIMIT: process.env.BUF_LIMIT || 10000 // 10000 lines
-, COMPRESS: isNaN(process.env.COMPRESS) ? true : +process.env.COMPRESS
+, COMPRESS: isNaN(process.env.COMPRESS) || process.env.COMPRESS === '1'
```

Comment:
is there any example for the possible values of `process.env.COMPRESS`? If not, can you add?

Reply:
so this is why we need the `config` map.  It's a boolean.  Where do you want me to add the possible values?

Reply:
I was thinking about adding examples to README.md if there is specific section for env variables but it seems there is not + this config part is gonna get rewritten, right?

Reply:
Yeah, we're gonna re-write all of this.  And the docs are severely lacking now, which is why I hope the config package will clear lots of this up (because it generates docs)